src/main/scala/firrtl/passes/Passes.scala

```diff
@@ -602,73 +602,63 @@ object InferWidths extends Pass {
 }
 
 object PullMuxes extends Pass {
-   private var mname = ""
    def name = "Pull Muxes"
    def run (c:Circuit): Circuit = {
       def pull_muxes_e (e:Expression) : Expression = {
          val ex = e map (pull_muxes_e) match {
-            case (e:WRef) => e
-            case (e:WSubField) => {
-               e.exp match {
-                  case (ex:Mux) => Mux(ex.cond,WSubField(ex.tval,e.name,e.tpe,e.gender),WSubField(ex.fval,e.name,e.tpe,e.gender),e.tpe)
-                  case (ex:ValidIf) => ValidIf(ex.cond,WSubField(ex.value,e.name,e.tpe,e.gender),e.tpe)
-                  case (ex) => e
-               }
+            case (e:WSubField) => e.exp match {
+               case (ex:Mux) => Mux(ex.cond,
+                  WSubField(ex.tval,e.name,e.tpe,e.gender),
+                  WSubField(ex.fval,e.name,e.tpe,e.gender),e.tpe)
+               case (ex:ValidIf) => ValidIf(ex.cond,
+                  WSubField(ex.value,e.name,e.tpe,e.gender),e.tpe)
+               case (ex) => e
             }
-            case (e:WSubIndex) => {
-               e.exp match {
-                  case (ex:Mux) => Mux(ex.cond,WSubIndex(ex.tval,e.value,e.tpe,e.gender),WSubIndex(ex.fval,e.value,e.tpe,e.gender),e.tpe)
-                  case (ex:ValidIf) => ValidIf(ex.cond,WSubIndex(ex.value,e.value,e.tpe,e.gender),e.tpe)
-                  case (ex) => e
-               }
+            case (e:WSubIndex) => e.exp match {
+               case (ex:Mux) => Mux(ex.cond,
+                  WSubIndex(ex.tval,e.value,e.tpe,e.gender),
+                  WSubIndex(ex.fval,e.value,e.tpe,e.gender),e.tpe)
+               case (ex:ValidIf) => ValidIf(ex.cond,
+                  WSubIndex(ex.value,e.value,e.tpe,e.gender),e.tpe)
+               case (ex) => e
```

Comment:
In generally when rewriting code I think we should try to replace things with more idiomatic style. There are some things that are more arguable than others, but at least putting spaces after commas would be nice.